tests/ts_updown_util.erl

```diff
@@ -235,11 +197,11 @@ query_with_client(Query, Node, Config) ->
     case Version of
         current ->
             riakc_ts:query(Client, Query);
-        previous ->
-            rpc:call(
-              ?CFG(previous_client_node, Config),
-              riakc_ts, query, [Client, Query])
-    end.
+                              previous ->
```

Comment:
Unindent me please. (If you update emacs Erlang mode to the current version, it will stop treating `query` as a keyword, and will autoindent normally.)